ui/src/routes.ts

```diff
@@ -42,13 +44,27 @@ const routes: IRoute[] = [
     enabled: true,
     component: CreateAuctionPage,
   },
+  {
+    key: 'edit-auction',
+    title: 'Edit auction',
+    path: '/edit-auction/:auctionId',
```

Comment:
replace to follow our routing style like this:
`path: '/auction/:auctionId/add-auction-item',`